app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,63 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun UserScoreProgressBar(
+    progress: Float,
+    modifier: Modifier = Modifier
+) {
+    Box(modifier = modifier
+        .size(40.dp)
+        .padding(2.dp)){
+        Canvas(modifier = Modifier.size(40.dp)) {
+
+            drawArc(
+                Color(red=212, green =255,blue=212),
```

Comment:
If you introducing some new color, please add it in `Color.kt` file, and than call it from there